keyboards/kbdfans/kbd67/rev2/keymaps/tucznak/rules.mk

```diff
@@ -0,0 +1,82 @@
+# MCU name
```

Comment:
This file only needs to contain the rules that you have changed from the keyboard-level `rules.mk` file (`keyboards/kbdfans/kbd67/rev2/rules.mk`).

This entire file can be collapsed down to:

```makefile
MOUSEKEY_ENABLE = no         # Mouse keys(+4700)
CONSOLE_ENABLE = no          # Console for debug(+400)
NKRO_ENABLE = yes            # USB Nkey Rollover
```

Though if you want, you could collapse it down to only the Build Options: https://github.com/qmk/qmk_firmware/blob/c296af50a3a7a87864988fdb42e1432a2abc4057/keyboards/kbdfans/kbd67/rev2/keymaps/tucznak/rules.mk#L61-L80

This has the slight advantage of keeping your options the same in the event anything changes at the keyboard level.